src/maintainer/conda_forge_yml.rst

```diff
@@ -226,7 +228,7 @@ modified.  Tools like conda-smithy may modify this, as needed.  It has a single
 
 .. code-block:: yaml
 
-    appveyor:
+    circle:
       secure:
         BINSTAR_TOKEN: <some big hash>
```

Comment:
I think we are removing this block in other PR so this change might be moot.

See https://github.com/conda-forge/conda-forge.github.io/pull/1793